src/ProgressMeter.jl

```diff
@@ -47,6 +48,9 @@ function BarGlyphs(s::AbstractString)
     return BarGlyphs(glyphs...)
 end
 
+
+is_logging(io) = isa(io, Base.TTY) == false || (get(ENV, "CI", nothing) == "true")
```

Comment:
I don't think this function should reside in this package, but it could be given as an example.

Reply:
resolved in 3cab602